terraform/function.tf

```
@@ -0,0 +1,69 @@
+data "archive_file" "source" {
+  type        = "zip"
+  source_dir  = "../"
```

Use `${path.module}` variable to avoid an invalid file when running in a different cwd.

```suggestion
  source_dir  = "${path.module}/../"
```

<https://developer.hashicorp.com/terraform/language/expressions/references#path-module>